gateway/1.5/modules/ROOT/pages/policies-pdk-create-project.adoc

```diff
@@ -2,4 +2,123 @@
+| `--version=<value>` | `1.0.0` | No | Version of the new policy. +
+The version format is `<major-release>.<minor-release>.<patch-release>`.
+|===
+
+== Setup a New Policy Project
+
+If you haven't run the `setup` command for a prior project, you must run the `setup` command from the policy's root directory after you create a custom policy project:
+
+Run the setup command:
+
+[source,ssh]
+----
+make setup
+----
+
+The setup command installs the PDK project's dependencies. It is only necessary to run this command if this is the first time you are creating a project on this device.
+
+If prompted, provide your Anypoint Platform credentials. 
+
+== Project Structure
+
+After running the setup command, the policy's root directory contains the following directories and files shown in the directory tree: 
+
+----
+├─ definition/  # Contains the files that describe the Policy Definition
+│  ├── target/  # Contains the build definition asset files
+│  └── gcl.yaml # GCL file managed by the developer that describes the Policy Definition
+├─ target/      # Contains the implementation asset files and the output of the policy build
+├─ src/         # Contains the implementation source code
+├─ test/        # Contains the artifacts required for running the policy locally
+├─ Makefile     # Provides all the callable actions required during the policy development lifecycle
+├─ Cargo.toml   # Rust language standard file that contains the metadata required to build the policy
+└─.project.yaml # Project manifest used by PDK
+----
+
+=== target/ and definition/target/
+
+There are two separate `target` directories:
+
+* `definition/target/`: contains the policy asset definition published to exchange
+* `target/`: Contains the files for the Policy asset implementation and the output of the Rust compilation process
+
+Both `target` directories and their content are included in the `.gitignore` file. You must compile the targets on each individual devices. For example, if you are developing a policy as part of a team, your team can publish the policy and it's source code to a remote repository, but each team member must build the target on their device for local testing.
+
+=== definition/
+
+The `definition` directory contains the `gcl.yaml` file that defines the policy's metadata and the configuration properties.
+
+To configure the `gcl.yaml` file, see xref:policies-pdk-create-schema-definition.adoc[].
+
+=== Cargo.toml
+
+Custom policies for Flex Gateway are developed in the Rust Programming language and compiled to a WebAssembly binary. 
+
+The `Cargo.toml` is Rust project manifest. Its `[package]` section contains the basic information of the policy implementation. The version specified in `Cargo.toml` matches the policy asset version. You can manage the policy version by editing the version number in `Cargo.toml`. The version follows the Semantic Versionsing (SemVer) format: `<major-version>.<minor-version>.<patch-version>`.
```

Comment:
> The version follows the Semantic Versionsing (SemVer) format: `<major-version>.<minor-version>.<patch-version>`
This is not entirely accurate. SemVer permits some formats we do not support, for example: `1.0.1-integration`. The previous version is valid in SemVer, but not valid for us. We only admit the 3 numbered parts of the version: `1.0.1`, `1.2.3`, etc...